proxy/http2/Http2Stream.h

```diff
@@ -153,7 +153,7 @@ class Http2Stream : public ProxyClientTransaction
   ssize_t client_rwnd;
   ssize_t server_rwnd = Http2::initial_window_size;
 
-  LINK(Http2Stream, link);
+  // LINK(Http2Stream, link);
```

Comment:
Please remove unused code. BTW, I'm surprised that `Continuation` has LINK macro and we don't need this :D